public/visualization_fn.ts

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { get } from 'lodash';
+import { i18n } from '@kbn/i18n';
+import { PersistedState } from '../../../src/plugins/visualizations/public';
+import { ExpressionFunctionDefinition, Render } from '../../../src/plugins/expressions/public';
+import { getIndexPatterns, getFilterManager, getSearchService, getVisualization } from './services';
+import { enhancedTableRequestHandler } from './data_load/enhanced-table-request-handler';
+import { enhancedTableResponseHandler } from './data_load/enhanced-table-response-handler';
+import { documentTableResponseHandler } from './data_load/document-table-response-handler';
```

Comment:
could you merge these 2 lines in only one line?

Reply:
Those two functions come from two different files, so I don't think I can merge them in just one line. Or maybe there is a way to do that merge that I don't know... I'm quite unexperienced with typescript :P

Reply:
Yes, you're right, I misread the two lines :)